.gitignore

```diff
@@ -2,6 +2,7 @@ HELP.md
 
 ### Gradle ###
 .gradle
+../.gradle
```

Comment:
Why do you need to ignore something in a parent dir. I didn't think that was how `.gitignore` worked.

Reply:
No idea. I removed it.